docs/pages/repo/docs/getting-started/existing-monorepo.mdx

```diff
@@ -41,6 +41,8 @@ Specify your `packages` in `pnpm-workspace.yaml`.
 packages:
   - "packages/*"
   - "apps/*"
+  // Note that the below is NOT supported
+  - "nested-workspaces/**"
```

Comment:
I think we should make this a `<Callout>` and say something more verbose about [`pnpm` configuration](https://pnpm.io/pnpm-workspace_yaml). 

We can say something like:

"pnpm supports the ability [declare workspaces in subdirectories](https://pnpm.io/pnpm-workspace_yaml) with a `**`. This effectively means that with pnpm you can have nested workspaces. Turborepo does not support this at the moment."

Reply:
👍🏼 SGTM